CHANGELOG.md

```diff
@@ -11,17 +11,32 @@ for Rust libraries in [RFC #1105](https://github.com/rust-lang/rfcs/blob/master/
 * The `.for_update()` method has been added to select statements, allowing
   construction of `SELECT ... FOR UPDATE`.
 
-* Added `insert(&default_values())` as a replacement for
+* Added `insert_into(table).default_values()` as a replacement for
   `insert_default_values()`
 
+* Added `insert_into(table).values(values)` as a replacement for
+  `insert(values).into(table)`.
+
+* Added support for MySQL's `REPLACE INTO` as `replace_into(table)`.
+
+* Added `replace_into(table).values(values)` as a replacement for
```

Comment:
Why isn't you commit talking about that ? Should this have been in another PR ?

Reply:
Given that `insert_or_replace` is exactly analogous to `insert`, it made sense to do them at the same time. While working on it, I noticed that having it be `INSERT OR REPLACE` was unnecessary, and it could just be `REPLACE` which is compatible with MySQL.